integrations/pagerduty/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+## 0.1.121 (2024-11-25)
+
+
+### Improvements
+
+- Improved on the integration default resources for PagerDuty incidents kind
+- Added blueprint and mapping for PagerDuty users
+- Added logs to track the HTTP requests made to PagerDuty API
+- Added a generic resync event handler to allow the integration to bring any valid resource from PagerDuty
```

Comment:
which ones are expected to work? can we make sure we have a proper log on when we aren't able to sync data for the passed "generic" kind?